fetcher/utils.go

```diff
@@ -39,3 +39,19 @@ func parseToGauge(gauge prometheus.Gauge, value string) {
 	}
 	gauge.Set(parsed)
 }
+
+func parseAdditionalLabels(additionalLabels []string, labels map[string]string) (result []string) {
```

Comment:
Very nice! At least for me, this is much clearer. I know that this might very well just be my personal taste though 😉. Thank you!